homeassistant/components/deconz/config_flow.py

```diff
@@ -58,6 +59,11 @@ class DeconzFlowHandler(ConfigFlow, domain=DOMAIN):
 
     _hassio_discovery: dict[str, Any]
 
+    bridges: list[DiscoveredBridge]
+    host: str
```

Comment:
Given these aren't always set, would it be more correct to say `host: str | None = None` (and similar for the other instance variables)? This might need checks for them being not `None` in the methods where there used but not set (e.g. `_create_entry`).

Reply:
I did it this way because all first uses of the Properties is to assign. If you think it is more correct to change to this I can change it.

Reply:
Yeah, on close examination it looks like the only function that reads without setting them is `_create_entry`, which is always called after the values are set.

Reply:
I'm ok with this, but I think it might need examination from one of the core maintainers to check it follows their standards.

Reply:
Thanks @chishm! I appreciate your effort here!